README.md

```diff
@@ -1,27 +1,36 @@
 # MLBox
 
-This is the MLBox Prototype. This is  still under construction, some parts probably don't work yet, or may have unexpected/inconsistent behaviours.
```

Comment:
IMHO this change can be removed helps in tracking README changes in future .
Readme changes are not required for mkdocs PR.

Reply:
Addressed